go.mod

```diff
@@ -103,7 +103,7 @@ require (
 	github.com/stretchr/testify v1.8.4
 	github.com/tdakkota/asciicheck v0.2.0
 	github.com/tiancaiamao/appdash v0.0.0-20181126055449-889f96f722a2
-	github.com/tikv/client-go/v2 v2.0.8-0.20231219052137-6f9ba8327b75
+	github.com/tikv/client-go/v2 v2.0.8-0.20231220023527-4ce1e45671b3
```

Comment:
The client-go version is ahead of this. pls merge master.

Reply:
done